assets/Product.wxs

```diff
@@ -10,14 +10,16 @@
   <?define ProductName = "$(env.ProductName)" ?>
   <?define ProductGuid = "$(env.ProductGuid)" ?>
   <?define ProductVersion = "$(env.ProductVersion)" ?>
+  <?define ProductSemanticVersion = "$(env.ProductSemanticVersion)" ?>
   <?define ProductVersionWithName = "$(var.ProductName)_$(var.ProductVersion)"?>
+  <?define ProductSemanticVersionWithName = "$(var.ProductName)_$(env.ProductSemanticVersion)"?>
```

Comment:
I used underscore to separate Product name from version (Ex: PowerShell_6.0.0-alpha.17)

Since we already have  a hyphen in the semantic version, used underscore to logically separate the two pieces.

Also, for some configurations in wxs, WiX does not like spaces. So went with underscore approach